src/autodiff.jl

```diff
@@ -38,6 +38,17 @@ function _init_seed!(t1sseeds, coloring, ncolor, nmap)
     end
 end
 
+function transfer!(target::AbstractJacobian, origin::AbstractJacobian)
```

Comment:
Idem: how about renaming this function `Base.copy!`?

Reply:
It's not really a copy though. Only the relevant parts for the computation are transferred. Not everything is copied like the mapping.